modules/ossm-troubleshooting-operators.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-operators_{context}"]
+= Troubleshooting service mesh Operators
+
+If you experience Operator issues, verify your Operator subscription status.  Also verify that you did not inadvertently install a community Operator.
+
+The Operator installation steps include verifying the Operator status in the OpenShift console.  You can also verify the Operator pods are available and running in the `openshift-operators` namespace with the following command:
```

Comment:
To troubleshoot something you have first to understand what this something does. I am missing a link to what the Service Mesh Operator does in a high level view, and then point out what are the possible issues and its side effects that each step of this high level view. Only looking at the logs we are relying entirely in the quality of the log messages only (which we also target to to high quality info), and even though ignoring the fact that understand all steps the operator does to have the Service Mesh setup is crucial to the end user attempt to solve the problem. If there's no plan now to explain it to the user, at least we have to track this requirement.

Reply:
@tvieira Good point.  Could you create a ticket for that and attach it to the troubleshooting Epic (https://issues.redhat.com/browse/OSSMDOC-253)?  This is only the skeleton of the Troubleshooting info, there's more to come.

Reply:
Done - https://issues.redhat.com/browse/OSSMDOC-294 Thanks @JStickler !